en/manage-cli-interfaces.md

```diff
@@ -0,0 +1,223 @@
+Title: CLI Interface Management
+table_of_contents: True
+
+
+# CLI Interface Management
+
+This is a list of interface management tasks to perform with the MAAS CLI. See
+[MAAS CLI][manage-cli] on how to get started.
+
+See [Networking][networking] for an overview of networking and 
+[Commission Nodes][commission-nodes] for more details on managing interfaces
+from the MAAS web UI.
+
+
+## Interface identifiers
+
+A numeric interface identifier is used by the MAAS CLI for many interface
+operations. Use the following command to retrieve the identifier(s):
+
+```bash
+maas $PROFILE interfaces read $SYSTEM_ID
+```
+
+Look for either *id* or the number at the end of an interface's resource URI,
+such as **15** in the following example output:
+
+```json
+"id": 15,
+"mac_address": "52:54:00:55:06:40",
+...
+"name": "ens9",
+...
+"resource_uri": "/MAAS/api/2.0/nodes/4efwb4/interfaces/15/"
+```
+
+## Create a Bond interface
+
+A bond interface is used to aggregate two or more more physical interfaces into
+a single logical interface. A bond can be created with the following command:
+
+```bash
+maas $PROFILE interfaces create-bond $SYSTEM_ID name=$BOND_NAME \
+parents=$IFACE1_ID mac_address=$MAC_ADDR \ 
+parents=$IFACE2_ID bond_mode=$BOND_MODE \
+bond_downdelay=$BOND_DOWN bond_updelay=$BOND_UP mtu=$MTU
+```
+
+Use the 'parents' parameters to define which interfaces form the aggregate
+interface. The following is an example of 'create-bond' in action:
+
+```bash
+maas admin interfaces create-bond 4efwb4 name=bond0 parents=4 \
+mac_address=52:52:00:00:00:00 parents=15 bond_mode=802.3ad bond_downdelay=200 \
+bond_updelay=200 mtu=9000
+```
```

Comment:
I know the bond modes are explained elsewhere - is it worth linking to that?

Reply:
There is - I added the details to the web UI section and put the link just beneath here.